service/src/main/java/bio/terra/policy/service/policy/PolicyGroupConstraint.java

```diff
@@ -29,6 +29,11 @@ public PolicyName getPolicyName() {
    */
   @Override
   public PolicyInput combine(PolicyInput dependent, PolicyInput source) {
+    if (source == null || dependent == null) {
```

Comment:
I don't understand this. If source is not null and dependent is null, why is the combination the dependent?

Reply:
Maybe I should make it more explicit for clarity. If source is null then return dependent is pretty clear. But also, if dependent is null and we can't modify it in M1 then we return null (equal to dependent) regardless of whatever source has.

Reply:
or maybe source = null and dependent = null, but that shouldn't come through the combiner, right?

Reply:
OK, I see. Thanks